test/intl402/Segmenter/iterator/branding.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-segment-iterator-prototype
+description: Verifies the behavior for the iterators.
+features: [Intl.Segmenter]
+---*/
+
+const otherReceivers = [
+    1, 123.45, undefined, null, "string", true, false,
+    Intl, Intl.Segmenter, Intl.Segmenter.prototype,
+    prototype,
+    new Intl.Segmenter(),
+    new Intl.Collator(),
+    new Intl.DateTimeFormat(),
+    new Intl.NumberFormat(),
+];
+for (const rec of otherReceivers) {
+   assert.throws(TypeError, () => prototype.next.call(rec));
```

Comment:
I don't think this is right. What is "prototype" here?

Reply:
I got "ReferenceError: prototype is not defined" in v8 instead because prototype is undefined. I believe Ms2ger forget to include some code in branding.js

Reply:
@Ms2ger

Reply:
@littledan  @gsathya

Reply:
Yes, definitely looks like a bug. We'll make sure to fix this PR soon. Until then, it's good to hold off on merging it.

Reply:
Agree!